packages/examples/create-react-app/src/components/Grid.tsx

```diff
@@ -20,6 +20,20 @@ const entities = [
     { id: 'D', src: 'nazi.png', position: { x: 2, y: 3 }, direction: -1 },
     { id: 'E', src: 'nazi.png', position: { x: 12, y: 33 }, direction: -1 },
     { id: 'F', src: 'nazi.png', position: { x: 4, y: 34 }, direction: -1 },
+    { id: 'G', src: 'player.png', position: { x: 10, y: 10 }, direction: -1 },
+    { id: 'H', src: 'player.png', position: { x: 20, y: 20 }, direction: -1 },
+    { id: 'I', src: 'player.png', position: { x: 30, y: 30 }, direction: -1 },
+    { id: 'G', src: 'nazi.png', position: { x: 2, y: 3 }, direction: -1 },
```

Comment:
bro what is this lol

Reply:
Lol, it's the nazi from castle Wolfenstein, the demo is top down wolfenstein